docs/osquery/osquery.asciidoc

```diff
@@ -121,9 +121,12 @@ image::images/scheduled-pack.png[Shows queries in the pack and details about eac
 
 [float]
 [[osquery-manage-query]]
-== Edit saved queries
+== Saved queries
```

Comment:
Should this instead be "Save queries" so that the wording is parallel with the rest of the headings?